### PR TITLE
add arm64 build

### PR DESCRIPTION
**Pull request by wfurt**
This adds arm64 windows binaries. I originally tried it as separate pipeline but that has two problems:
- we don't have any machines in pool
- and it complicates packaging as we would need to consolidate artifacts from multiple builds for the final package assembly.

so instead this uses the existing hack for x86 when we simply build all three binaries unconditionally for Windows and we package them into Nuget in single step. 

Since this repro is primarily used as transport I feel this is reasonable compromise. 
if build by developers locally, it will require ARM targets in VS (probably already added to build runtime) 

contributes to https://github.com/dotnet/runtime/issues/82885

### CODE REVIEW

File: src/System.Net.MsQuic.Transport/System.Net.MsQuic.Transport.csproj [hunk 1]
```diff
@@ -26,6 +26,13 @@
             <TfmSpecificPackageFile
                 Include="..\msquic\artifacts\bin\windows\x64_$(Configuration)_schannel\msquic.pdb"
                 PackagePath="runtimes/win10-x64/native/"/>
+
+            <TfmSpecificPackageFile
+                Include="..\msquic\artifacts\bin\windows\arm64_$(Configuration)_schannel\msquic.dll"
+                PackagePath="runtimes/win10-x64/native/"/>
```
Comment (ManickaP) [inline src/System.Net.MsQuic.Transport/System.Net.MsQuic.Transport.csproj:32]: Typo ?
```suggestion
                PackagePath="runtimes/win10-arm64/native/"/>
```

File: src/System.Net.MsQuic.Transport/System.Net.MsQuic.Transport.csproj [hunk 2]
```diff
@@ -26,6 +26,13 @@
...
+            <TfmSpecificPackageFile
+                Include="..\msquic\artifacts\bin\windows\arm64_$(Configuration)_schannel\msquic.pdb"
+                PackagePath="runtimes/win10-x64/native/"/>
```
Comment (ManickaP) [inline src/System.Net.MsQuic.Transport/System.Net.MsQuic.Transport.csproj:35]: Same as above.

File: src/System.Net.MsQuic.Transport/System.Net.MsQuic.Transport.csproj (hunk 1 above)
Comment (wfurt) [reply src/System.Net.MsQuic.Transport/System.Net.MsQuic.Transport.csproj]: good catch. updated.

**Review by hoyosjs — APPROVED** [review]

**Review by ManickaP — APPROVED** [review]
LGTM, thanks.
Do we also need arm 32 bit?

**Comment by wfurt** [comment]
We do not have Windows ARM32 AFAIK @manickap.

**Pull request merged**